ios/FluentUI/Core/Theme/FluentTheme.swift

```diff
@@ -90,7 +90,7 @@ extension UIWindow: FluentThemeable {
     }
 }
 
-@objc extension UIView {
```

Comment:
why is this pbulic now?

Reply:
This is so that we can access the `FluentTheme` from the view. This is change is also part of this [PR](https://github.com/microsoft/fluentui-apple/pull/1148/files#diff-856de3e2f4dd07c4817a2509df562f4e32fa86b0b346dcc64d9fb51b9097e5c7).

Reply:
It would be good if we had these in entirely separate changes - could you ensure you're merging this change in from that PR?